packages/ajv-validator/tsconfig.json

```diff
@@ -1,10 +1,6 @@
 {
   "extends": "../../tsconfig.json",
   "compilerOptions": {
-    "rootDir": "./src",
-    "outDir": "./dist",
-  },
-  "include": [
-    "src/**/*.ts"
-  ],
-}
```

Comment:
These options are both redundant, only `outDir` is really necessary

- [`rootDir`](https://www.typescriptlang.org/tsconfig/#rootDir)
- [`include`](https://www.typescriptlang.org/tsconfig/#include)